### PR TITLE
Rustup to rustc 1.61.0-nightly (d53246fed 2022-03-25)

### PR DESCRIPTION
**Pull request by bjorn3**

### CODE REVIEW

**Comment by antoyo** [comment]
Thanks!
There's an unexpected failure in the CI.

**Comment by antoyo** [comment]
Is this ready to be merged?

File: src/lib.rs [hunk 1]
```diff
@@ -101,14 +102,19 @@ impl CodegenBackend for GccCodegenBackend {
         *self.supports_128bit_integers.lock().expect("lock") = check_context.get_last_error() == Ok(None);
     }
 
+    fn provide(&self, providers: &mut Providers) {
+        // FIXME compute list of enabled features from cli flags
```
Comment (antoyo) [inline src/lib.rs:106]: You can add my name to that FIXME:

File: test.sh [hunk 2]
```diff
@@ -165,6 +165,24 @@ function test_rustc() {
...
+index 887d27fd6dca4..2c2239f2b83d1 100644
+--- a/src/tools/compiletest/src/header.rs
++++ b/src/tools/compiletest/src/header.rs
+@@ -806,8 +806,8 @@ pub fn make_test_description<R: Read>(
+     cfg: Option<&str>,
+ ) -> test::TestDesc {
+     let mut ignore = false;
+     #[cfg(not(bootstrap))]
+-    let ignore_message: Option<String> = None;
++    let ignore_message: Option<&str> = None;
```
Comment (antoyo) [inline test.sh:179]: Is this expected to require this patch?

**Review by antoyo — COMMENTED** [review]
Can you please revert #139 (issue https://github.com/rust-lang/rustc_codegen_gcc/issues/140)?

File: test.sh (hunk 2 above)
Comment (bjorn3) [reply test.sh:179]: Cg_clif needs the same patch. Probably some weirdness around bootstrapping that may solve itself on the next bootstrap bump.

File: src/lib.rs (hunk 1 above)
Comment (bjorn3) [reply src/lib.rs]: Done

**Comment by bjorn3** [comment]
Reverted #139.

**Pull request merged**